Makefile

```diff
@@ -313,19 +319,24 @@ list-eunit-suites:
 build-test:
 	@test/build/test-configure.sh
 
+ifeq ($(with_clouseau), 1)
+_WITH_CLOUSEAU="--with-clouseau"
+endif
 
 .PHONY: mango-test
 # target: mango-test - Run Mango tests
 mango-test: export COUCHDB_TEST_ADMIN_PARTY_OVERRIDE=1
 mango-test: devclean all
-	@cd src/mango && \
-		python3 -m venv .venv && \
-		.venv/bin/python3 -m pip install -r requirements.txt
-	@cd src/mango && \
-		../../dev/run "$(TEST_OPTS)" \
+	@python3 -m venv src/mango/.venv && \
```

Comment:
Only a small nit. I don't know if we used it consequently, but should we use the defined env var `PYTHON` here?

Reply:
Yes, I mean `Makefile.win`.

Reply:
All right, done.